google/google.go

```diff
@@ -44,6 +45,8 @@ func init() {
 	gob.Register(goauth.Userinfo{})
 }
 
+var loginURL string
```

Comment:
Are we fine with limitation that loginURL is a singleton for app?

What if semantic of Auth middleware takes this as param (e.g. `Auth(loginURL string)`?

Reply:
I think it's out of scope for the PR, but you are right we likely want to rewrite parts to enclose the data in a struct. Right now the global functions make it not possible to do this.

Reply:
I agree w/ you @fogfish - I can create a new issue ticket to spec out the rewrite if you all think that would be beneficial. I'm assuming this would require a major version increment too. Maybe it'd be worth creating a master ticket for v2 and see what type of traction or interest there is?

Reply:
I added https://github.com/zalando/gin-oauth2/issues/79 , one day I have to come back and do it :)